src/app/components/tv-show-table/tv-show-table.component.html

```diff
@@ -0,0 +1,37 @@
+<table>
+  <thead>
+  <tr>
+    <th>Name</th>
+    <th>Country</th>
+    <th>Year started</th>
+    <th>Status</th>
+    <th>Actions</th>
+  </tr>
+  </thead>
+  <tbody>
+    @defer(when fetchComplete){
```

Comment:
I'd reconsider this because lazy-loading those ten lines of HTML after the API is done will make things slower instead of faster. At the very least, I'd do it before fetch is complete.

Reply:
I have swapped out this @defer block for an @if block. Does this resolve this? OR are you suggesting I only display the data points when fetchComplete is true AND instead render the HTML for the table rows initially with "display: none?"

Reply:
That works.